src/symtable.js

```diff
@@ -422,8 +422,8 @@ SymbolTable.prototype.newTmpname = function (lineno) {
 SymbolTable.prototype.addDef = function (name, flag, lineno) {
     var fromGlobal;
     var val;
-    var mangled = Sk.mangleName(this.curClass, new Sk.builtin.str(name)).v;
-    mangled = Sk.fixReservedNames(mangled);
+    var mangled = Sk.mangleName(this.curClass, name).v;
```

Comment:
I’m fairly certain that the name object passed here is always a `Sk.builtin.str` object but someone who knows this part of the code base can conform -  @albertjan ? It seemed unnecessary then to str a str. I did put an assert statement when working on this pr and ran all the tests and there was no issue with this change...

Reply:
I mean the fact the tests pass mean that you're probably right. Would `mangleName` fail if it was called with a js string?

Reply:
```
function mangleName (priv, ident) {
    var name = ident.v;
```
 
Yeah it would. So you're fine.